storybook/storybook-react/src/Grid/Grid.docs.mdx

```diff
@@ -61,4 +53,47 @@ In sommige gevallen kan daar beter meer of minder witruimte staan.
 
 <Canvas of={GridStories.VerticalGap} />
 
-Voor meer voorbeelden raadpleeg je [Grid Cell](?path=/docs/react_layout-grid-cell--docs) zelf.
+### Cellen beslaan kolommen
+
+Een cel beslaat standaard 1 kolom in het grid.
+Om een cel meer kolommen te laten beslaan, gebruik je de `span` prop.
+
+<Canvas of={GridStories.CellSpan} />
+
+### Verschillende breedtes
+
+Je kunt het aantal kolommen dat een cel beslaat laten afhangen van de vensterbreedte.
+Gebruik hiervoor de `span` prop met 3 waarden voor respectievelijk smalle, middelbrede en brede vensters.
+
+Via `span={{ narrow: 4, medium: 6, wide: 8 }}` is deze cel 4 van de 4 kolommen breed op smalle vensters, 6 van de 8 op middelbrede en 8 van de 12 op brede vensters:
+
+<Canvas of={GridStories.CellSpans} />
+
+### Volledige breedte
+
+Om de cel de volledige breedte te geven – of het grid nu 4, 8, of 12 kolommen heeft – gebruik je de `fullWidth` prop.
+
+<Canvas of={GridStories.FullWidth} />
+
+### Startpositie
+
+Elke cel start automatisch in de eerstvolgende beschikbare positie in het grid.
+Een waarde van 1 hoef je dus meestal niet expliciet op te geven.
+Je kunt de startpositie van een cel aanpassen met de `start` prop.
+Zo kun je bijvoorbeeld cellen in meerdere rijen met elkaar uitlijnen.
+Of een cel in het midden van een breed grid plaatsen.
+Ook komt dit voor als een kolom als extra witruimte wilt gebruiken.
+
+Het kan ook voorkomen dat ook de startpositie van een cel afhangt van de vensterbreedte.
+Gebruik hiervoor de `start` prop met 3 waarden, precies zoals bij `span`.
+
+Een voorbeeld met `start={2}`:
+
+<Canvas of={GridStories.StartPosition} />
+
+## Richtlijnen
+
+Zorg ervoor dat het aantal kolommen dat je aan een cel toekent past bij het aantal kolommen van het grid voor de betreffende vensterbreedte.
+Datzelfde geldt voor het starten van een cel in een latere kolom.
+Als het totaal van beide waarden is te groot is voegt de browser kolommen toe aan het grid.
```

Comment:
Do we want to implement a check for this?

Reply:
I don’t think so.

Reply:
Why not? Would be doable, right? We're allowing users to break the 4, 8, 12 column grid now, which isn't ideal imo

Reply:
We'll make a seperate story for this